llvm/cmake/config-ix.cmake

```diff
@@ -257,6 +257,22 @@ else()
   set(LLVM_ENABLE_TERMINFO 0)
 endif()
 
+#Check for icu.
+find_package(ICU COMPONENTS uc i18n)
```

Comment:
Can we put this behind an option like `LLVM_ENABLE_ICU` (and similarly for iconv) akin to what we do for other dependencies such as zstd? ICU is a large dependency and some users may want to omit it even if it's available on their system (and conversely some vendors might want to fail the build if it's missing).

Reply:
Thanks, I've created LLVM_ENABLE_ICU and LLVM_ENABLE_ICONV, I've defaulted both to be ON